apps/src/applab/designElements/dropdown.jsx

```diff
@@ -88,6 +88,16 @@ module.exports = {
     element.style.height = '30px';
     element.style.fontSize = '14px';
     element.style.margin = '0';
+    element.style.color = '#fff';
+    element.style.backgroundColor = '#1abc9c';
```

Comment:
Also note that for these specific colors, they're associated with this Applab "theme" and not necessarily colors we want to use elsewhere on the site.  It's by-design that students app have a different look & feel from the rest of Code Studio.